### ISSUE TITLE
server version 0.00.000.00 is not supported

### ISSUE THREAD

**Issue by xtez** [SAP/go-hdb#60]
% cat main.go
```
package main

import (
        "database/sql"
        "log"

        "github.com/SAP/go-hdb/driver"
)

func main() {
        connector := driver.NewBasicAuthConnector("HOST_NAME:PORT", "USER", "PASSWORD")
        connector.SetTimeout(60)
        hanaDB := sql.OpenDB(connector)
        defer hanaDB.Close()

        if err := hanaDB.Ping(); err != nil {
                log.Fatal(err)
        }
}
```

% go run main.go 
```
2020/10/14 13:36:32 server version 0.00.000.00 is not supported
exit status 1
 ```

% cat go.mod 
```
module https://github.com/xtez/hana

go 1.15

require github.com/SAP/go-hdb v0.101.1
 ```

% cat go.sum
```
github.com/SAP/go-hdb v0.101.1 h1:+8zjTmPL6bt6FsG+bZZluh0jc86QfUIwaydMHaRVjuI=
github.com/SAP/go-hdb v0.101.1/go.mod h1:sfLJoH5jM9mbyEKfNCfq1jnSOi5qYPeR+hvJprEfvpk=
golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2/go.mod h1:djNgcEr1/C05ACkg1iLfiJU5Ep61QUkGW8qpdssI0+w=
golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9 h1:psW17arqaxU48Z5kZ0CQnkZWQJsqcURM6tKiBApRjXI=
golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9/go.mod h1:LzIPMQfyMNhhGPhUkYOs5KpL4U8rLKemX1yGLhDgUto=
golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3/go.mod h1:t9HGtf8HONx5eT2rtn7q6eTqICYqUVnKs3thJo3Qplg=
golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a/go.mod h1:STP8DvDyc/dI5b8T5hshtkjS+E42TnysNCUPdjciGhY=
golang.org/x/sys v0.0.0-20190412213103-97732733099d/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
golang.org/x/text v0.3.0/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ=
golang.org/x/text v0.3.2 h1:tW2bmiBqwgJj/UpqtC8EpXEZVYOwU0yG4iWbprSVAcs=
golang.org/x/text v0.3.2/go.mod h1:bEr9sfX3Q8Zfm5fL9x+3itogRgK3+ptLWKqgva+5dAk=
golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e/go.mod h1:n7NCudcB/nEzxVGmLbDWY5pfWTLqBcC2KZ6jyYvM4mQ=
```

Additional Info:
```
SELECT VERSION FROM "SYS"."M_DATABASE";
VERSION
1.00.122.25.1559112397
```

**Comment by stfnmllr** [comment]
Yes, reporting 
`2020/10/14 13:36:32 server version 0.00.000.00 is not supported
`
when using HANA 1.00 is a known issue, which is fixed with go-hdb version v0.102.0.

But what might be more important is that the go-hdb did drop support for HANA server versions < 2.00.042 with go-hdb version v0.101.0 (please see release notes).
Anyway, I might try to add support of HANA version 1.00 SPS 12 again with v0.102.0 which is going to be released soon. So, I am going to close this issue as soon as the new go-hdb version is available.

**Comment by stfnmllr** [comment]
Can you please double check with the development branch
https://github.com/SAP/go-hdb/tree/develop
and report back if it is working?
Thanks!

**Comment by zhankenson** [comment]
> Yes, reporting
> `2020/10/14 13:36:32 server version 0.00.000.00 is not supported `
> when using HANA 1.00 is a known issue, which is fixed with go-hdb version v0.102.0.
> 
> But what might be more important is that the go-hdb did drop support for HANA server versions < 2.00.042 with go-hdb version v0.101.0 (please see release notes).
> Anyway, I might try to add support of HANA version 1.00 SPS 12 again with v0.102.0 which is going to be released soon. So, I am going to close this issue as soon as the new go-hdb version is available.

version 2.00.024.06.1538035880 is not supported

**Comment by stfnmllr** [comment]
> version 2.00.024.06.1538035880 is not supported，Will it be resolved in v0.102.0？

Yes, with the development branch respectively the version v.0.102.0  (to be released soon) it should work.

**Comment by zhankenson** [comment]
> should

ok,thanks

**Comment by zhankenson** [comment]
> > version 2.00.024.06.1538035880 is not supported，Will it be resolved in v0.102.0？
> 
> Yes, with the development branch respectively the version v.0.102.0 (to be released soon) it should work.

ok,thanks

**Comment by stfnmllr** [comment]
Should work again now. Version
https://github.com/SAP/go-hdb/releases/tag/v0.102.0
is released.
